src/ext/java/org/opentripplanner/ext/transmodelapi/model/plan/TripPatternType.java

```diff
@@ -145,14 +145,25 @@ public static GraphQLObjectType create(
                         .build())
                 .field(GraphQLFieldDefinition
                         .newFieldDefinition()
-                        .name("waitTimeAdjustedGeneralizedCost")
+                        .name("waitTimeOptimizedCost")
```

Comment:
This is a backwards incompatible change, we should create a new field, and deprecate this